test/e2e/basic_workflow_test.go

```diff
@@ -20,156 +20,146 @@ import (
 	"fmt"
 	"strings"
 	"testing"
-
-	"github.com/knative/client/pkg/util"
-	"gotest.tools/assert"
-)
-
-var (
-	e env
-	k kn
-)
-
-const (
-	KnDefaultTestImage string = "gcr.io/knative-samples/helloworld-go"
 )
 
-func Setup(t *testing.T) func(t *testing.T) {
-	e = buildEnv(t)
-	k = kn{t, e.Namespace, Logger{}}
-	CreateTestNamespace(t, e.Namespace)
-	return Teardown
-}
-
-func Teardown(t *testing.T) {
-	DeleteTestNamespace(t, e.Namespace)
-}
-
 func TestBasicWorkflow(t *testing.T) {
 	teardown := Setup(t)
 	defer teardown(t)
 
-	t.Run("returns no service before running tests", func(t *testing.T) {
-		testServiceListEmpty(t, k)
-	})
-
-	t.Run("create hello service and returns no error", func(t *testing.T) {
-		testServiceCreate(t, k, "hello")
-	})
-
-	t.Run("returns valid info about hello service", func(t *testing.T) {
-		testServiceList(t, k, "hello")
-		testServiceDescribe(t, k, "hello")
-	})
-
-	t.Run("update hello service's configuration and returns no error", func(t *testing.T) {
-		testServiceUpdate(t, k, "hello", []string{"--env", "TARGET=kn", "--port", "8888"})
-	})
-
-	t.Run("create another service and returns no error", func(t *testing.T) {
-		testServiceCreate(t, k, "svc2")
-	})
-
-	t.Run("returns a list of revisions associated with hello and svc2 services", func(t *testing.T) {
-		testRevisionListForService(t, k, "hello")
-		testRevisionListForService(t, k, "svc2")
-	})
-
-	t.Run("returns a list of routes associated with hello and svc2 services", func(t *testing.T) {
-		testRouteList(t, k)
-		testRouteListWithArgument(t, k, "hello")
-	})
-
-	t.Run("delete hello and svc2 services and returns no error", func(t *testing.T) {
-		testServiceDelete(t, k, "hello")
-		testServiceDelete(t, k, "svc2")
-	})
-
-	t.Run("returns no service after completing tests", func(t *testing.T) {
-		testServiceListEmpty(t, k)
-	})
+	testServiceListEmpty(t, k)
```

Comment:
What is the reason to revert this change we introduced lately ?

Reply:
Reverted. Interesting that GH or git did not complain when I pushed. Unless I somehow overwrote the files. Anyhow reverted and improved some more as I add another IT for plugin. I'll include that in the #249 which means this PR is a pre-req.